Example/Database/Tests/Unit/FLevelDBStorageEngineTests.m

```diff
@@ -473,7 +473,7 @@ - (void)testHugeLeafNodeThenDeeperSet {
 // It is kind of bad we raise "invalid" data, but at least we don't crash *trollface*
 - (void)testExtremeDoublesAsServerCache {
 #ifdef TARGET_OS_IOS
-  if ([[NSProcessInfo processInfo] operatingSystemVersion].majorVersion == 11) {
+  if ([[NSProcessInfo processInfo] operatingSystemVersion].majorVersion >= 11) {
```

Comment:
Wow, this was very forward thinking code when I originally wrote it *hides in shame*